lib/src/browser.dart

```diff
@@ -57,20 +61,23 @@ class Browser {
     _firefox,
     _safari,
     _internetExplorer,
+    _edge,
     _wkWebView
   ];
 
   bool get isChrome => this == _chrome;
   bool get isFirefox => this == _firefox;
   bool get isSafari => this == _safari;
   bool get isInternetExplorer => this == _internetExplorer;
+  bool get isEdge => this == _edge;
```

Comment:
support can always check isEdge for the canary tool to know for sure that it is Edge